dataprofiler/tests/labelers/test_integration_struct_data_labeler.py

```diff
@@ -323,7 +323,10 @@ def test_warning_tf_run_dp_multiple_times(self):
                 columns.append(col)
                 predictions.append(results['data_stats'][col]['data_label'])
 
-    def test_warning_tf_run_dp_merge(self):
+    from unittest import mock
+    @mock.patch('dataprofiler.profilers.profile_builder.'
+                'StructuredProfiler._merge_correlation')
+    def test_warning_tf_run_dp_merge(self, *mocks):
```

Comment:
Not that this is yours, but what is the intent of this test? it doesn't seem to assert anything. @granteden @ChrisWallace2020 ?

@AnhTruong why is the mock necessary?

Reply:
Maybe instead the correlations should be disabled like the other props below?

Reply:
similar statement to the tests below these in: `test_warning_tf_multiple_dp_with_update`

Reply:
changed, correlation option is disabled now